website/api/manager/ManagerAPI.php

```diff
@@ -81,20 +88,29 @@ protected function task() {
                 );
             }
 
+            // don't give out any task if games have been stopped.
+            if($this->checkConfig("compState", "noGameTasks")) {
+                return array("type" => "notask");
```

Comment:
but i'm being very nitpicky. just thought id note that the worker doesn't explicitly handle that.